pkg/config/config.go

```diff
@@ -160,11 +162,15 @@ func newProxyLocalConfig() proxyLocalConfig {
 	}
 }
 
+type ConfigUrl struct {
+	Name string
```

Comment:
please add yaml tags, so it shows with first capital letters, as the rest of the fields